middlewares/authentication.js

```diff
@@ -0,0 +1,16 @@
+const jwt = require('jsonwebtoken')
```

Comment:
@aapetsi The folder name is middleware on develop. This functionality also already exists in develop

Reply:
yes. i did that before Tolu pushed his up. I've fixed that conflict